version: "3.9"

services:
  grafana:
    image: grafana/grafana-oss:9.4.3
    container_name: grafana
    ports:
      - "53000:3000"
    volumes:
      - /media/psf/Docker/Grafana/conf/grafana.ini:/etc/grafana/grafana.ini:rw
      - /media/psf/Docker/Grafana/data:/var/lib/grafana:rw
    restart: on-failure
    networks:
      - develop
    environment:
      GF_INSTALL_PLUGINS: ${GF_INSTALL_PLUGINS}
      TZ: "Asia/Shanghai"

networks:
  develop:
    external: true